src/UNL/MediaHub/DefaultHomepage.php

```diff
@@ -22,12 +22,54 @@ function getCacheKey()
 
     function run()
     {
+        
+    }
+    
+    function getTopMedia()
+    {
         $options = array(
-            'limit'   => 6,
+            'limit'   => null,
             'orderby' => 'popular_play_count'
         );
-        $this->top_media = new UNL_MediaHub_MediaList($options);
-        $this->top_media->run();
+        $top_media = new UNL_MediaHub_MediaList($options);
```

Comment:
I guess I don't understand.  What run time filter?  The template is cached by savvy turbo and the `getTopMedia()` method is called from the template.